### ISSUE TITLE
Include guideline instruction link

### ISSUE THREAD

**Issue by mlewand** [cksource/ckeditor-plugin-a11ychecker#111]
We should include a link to more information about the problem if possible.

E.g. facing a failure F89 should result with adding a link to [F89 at WCAG2](http://www.w3.org/TR/WCAG-TECHS/F89.html).
